quickstart.sh

```diff
@@ -6,7 +6,7 @@ set -e
 # Variables
 : "${DOCKER_IMAGE_PARITY:=trustlines/tlbc-testnet}"
 DOCKER_IMAGE_WATCHTOWER="v2tec/watchtower"
-: "${DOCKER_IMAGE_QUICKSTART:=trustlines/quickstart:master0}"
+: "${DOCKER_IMAGE_QUICKSTART:=trustlines/quickstart:master3734}"
```

Comment:
Can't we use "latest" here?

Reply:
only if we stay compatible with what we have now forever (and if we would upload a latest image).